community/index.adoc

```diff
@@ -34,6 +35,7 @@ After the documentation, probably the best place to look for answers. We activel
 <a class="ui label" href="https://stackoverflow.com/questions/tagged/hibernate-validator">hibernate-validator</a>
 <a class="ui label" href="https://stackoverflow.com/questions/tagged/hibernate-reactive">hibernate-reactive</a>
 <a class="ui label" href="https://stackoverflow.com/questions/tagged/hibernate-tools">hibernate-tools</a>
+<a class="ui label" href="https://stackoverflow.com/questions/tagged/mongo-hibernate">mongo-hibernate</a>
```

Comment:
I'd remove "we actively monitor the following tags" above, because I don't think that's true for `hibernate`, and anyway the "we" is getting confusing now that we have `mongo-hibernate` here.